chapters/overloaded.tex

```diff
@@ -139,15 +139,15 @@ \section{Overloaded Constructors}\label{overloaded-constructors}
   The operator C.'constructor' shall only contain functions that declare
   one output component, which shall be of the operator record class C.
 \item
-  For an operator recordclass there shall not exist any potential call
+  For an operator record class there shall not exist any potential call
   that lead to multiple matches in (1) above.
   \begin{nonnormative}
   How to verify this is not specified.
   \end{nonnormative}
 \item
   For a pair of operator record classes \lstinline!C! and \lstinline!D! and components \lstinline!c! and \lstinline!d!
-  of these classes both of \lstinline!C.'constructor'(d)! and \lstinline!D.'constructor'(c)!
-  shall not both be legal.
+  of these classes at most one of \lstinline!C.'constructor'(d)! and \lstinline!D.'constructor'(c)!
```

Comment:
This is one of the many places where I suppose @GallLeo's grammar check would suggest to use a comma after _of these classes_.
```suggestion
  of these classes, at most one of \lstinline!C.'constructor'(d)! and \lstinline!D.'constructor'(c)!
```